<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupLogOptionsWidget</class>
 <widget class="QWidget" name="BackupLogOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Backup log</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_30">
   <item>
    <widget class="QFrame" name="frmBackupLog">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblBackupLog">
        <property name="text">
         <string>Backup log:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbBackupLogType">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Database</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBackupLogMode">
        <property name="text">
         <string>Default logging mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbBackupLogMode">
        <item>
         <property name="text">
          <string>save information about every performed test (Full)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status changes (Brief)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status changes + midnight logging (Midnight)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status or reply value changes (Reply)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stwBackupLog">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageBackupLogFile">
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <item>
        <widget class="QGroupBox" name="boxBackupLogFile">
         <property name="title">
          <string> File log  </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QComboBox" name="cmbBackupLogFile">
            <item>
             <property name="text">
              <string>Log to specific file</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every month</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every week</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every day</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editBackupLogFileName"/>
            </item>
            <item>
             <widget class="QToolButton" name="btnBackupLogDir">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../qHostMonitor.qrc">
                <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxBackupLogFormat">
         <property name="title">
          <string> Log format  </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_26">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbBackupLogHTML">
            <property name="text">
             <string>HTML file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbBackupLogText">
            <property name="text">
             <string>Text file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbBackupLogDBF">
            <property name="text">
             <string>DBF file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageBackupLogODBC">
      <layout class="QVBoxLayout" name="verticalLayout_29">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="boxBackupLogODBC">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string> ODBC log  </string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_29">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="lblBackupLogDatasource">
              <property name="text">
               <string>Data source</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbBackupLogDatasource">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_28">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_28">
                <item>
                 <widget class="QLabel" name="lblBackupLogSQL">
                  <property name="text">
                   <string>SQL query</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btnBackupLogSqlEdit">
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../qHostMonitor.qrc">
                    <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnBackupLogOdbcTest">
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QPlainTextEdit" name="plainBackupLogSQL">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="plainText">
               <string>Insert into hmlog (eventtime, testname, status, reply, testid, testmethod) VALUES ('%DateTime%', '%TestName%', '%Status%', '%Reply%', %TestID%, '%TestMethod%')</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="lblBackupLogLogin">
              <property name="text">
               <string>Login</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="editBackupLogLogin"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblBackupLogPassword">
              <property name="text">
               <string>Password</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="editBackupLogPassword"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblBackupLogTimeout">
              <property name="text">
               <string>Timeout</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_27">
              <item>
               <widget class="QSpinBox" name="spinBackupLogTimeout">
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblBackupLogTimeoutSec">
                <property name="text">
                 <string>sec</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmBackupLogAlerts">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkBackupLogAlertInaccessible">
        <property name="text">
         <string>Execute alert profile when log inaccessible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbBackupLogAlertInaccessible">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBackupLogAlertInaccessible">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Alert</string>
        </property>
        <property name="icon">
         <iconset resource="../../qHostMonitor.qrc">
          <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
